src/v0/destinations/tiktok_ads/transformV2.test.js

```diff
@@ -0,0 +1,25 @@
+const { processRouterDest } = require('./transformV2');
```

Comment:
you could use component framework for this

Reply:
There need mocking for `maxBatchSizeV2` in config.js I tried to do it taking reference from[ tradedesk PR](https://github.com/rudderlabs/rudder-transformer/pull/2918) but it doesn't seem to work

Reply:
done